src/context.tsx

```diff
@@ -41,147 +50,124 @@ export function getDefaultFilters(): Filters {
     specimen_type: new Set(),
     publish_date: new Set([new Date(2019, 1, 1, 1), new Date()]),
     estimate_grade: new Set([
-      Translate('EstimateGradeOptions', ['National']),
-      Translate('EstimateGradeOptions', ['Regional']),
-    ])
-  }
+      Translate("EstimateGradeOptions", ["National"]),
+      Translate("EstimateGradeOptions", ["Regional"]),
+    ]),
+  };
 }
 
 // Note: filters = elements that user has chosen to filter by
 // filter_options = all the elements that users could filter by
 const initialState: State = {
-  healthcheck: '',
   chartAggregationFactor: AggregationFactor.country,
   explore: {
     filters: getEmptyFilters(),
     records: [],
     estimateGradePrevalences: [],
-    metaAnalyzedRecords: [],
-    isLoading: false
+    isLoading: false,
   },
   // TODO: replace this with an obj
   // representing the data page once
   // we put filters there
-  analyze: {
-    filters: getDefaultFilters(),
-    records: [],
-    estimateGradePrevalences: [],
-    metaAnalyzedRecords: [],
-    isLoading: false
-  },
   allFilterOptions: getEmptyFilters(),
   dataPageState: {
     exploreIsOpen: true,
-    showStudiesModal: false,
-    routingOccurred: false
   },
   calendarStartDates: {
     // Important, the fact that we use an hour here tells us that we are using a default value
     minDate: new Date(2019, 1, 1, 1),
```

Comment:
Is it a bad idea to use the `minDate` and `maxDate` from context for `publish_date` in these two functions:

**getEmptyFilters()**
![image](https://user-images.githubusercontent.com/33983945/111040919-9ab12100-8403-11eb-995f-4d1fd3b7cccb.png)

**getDefaultFilters()**
![image](https://user-images.githubusercontent.com/33983945/111040927-a56bb600-8403-11eb-8194-109ef7a9b0d0.png)

Reply:
I'm not opposed, makes it a bit more readable